tests/tests/datm_ocn_ice_mx100_cfsr

```diff
@@ -0,0 +1,23 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_COLD test
```

Comment:
CFSR forcing.  No where in the name of the test, this is mentioned.

Reply:
Done.